packages/taco/integration-test/sign.test.ts

```diff
@@ -0,0 +1,78 @@
+import { Domain, getPorterUris } from '@nucypher/shared';
+import { ethers } from 'ethers';
+import { beforeAll, describe, expect, test } from 'vitest';
+
+import { initialize } from '../src';
+import { signUserOp } from '../src/sign';
+import { UserOperation } from '../src/types';
+
+const RPC_PROVIDER_URL = 'https://rpc-amoy.polygon.technology';
+const SIGNER_PRIVATE_KEY = '0x900edb9e8214b2353f82aa195e915128f419a92cfb8bbc0f4784f10ef4112b86';
+const DOMAIN = 'lynx';
+const RITUAL_ID = 27;
+const CHAIN_ID = 80002;
+
+describe('Taco Sign Integration Test', () => {
+  let provider: ethers.providers.JsonRpcProvider;
+  let signer: ethers.Wallet;
+
+  beforeAll(async () => {
+    provider = new ethers.providers.JsonRpcProvider(RPC_PROVIDER_URL);
+    signer = new ethers.Wallet(SIGNER_PRIVATE_KEY, provider);
+
+    // Initialize the library
+    await initialize();
+
+    // Verify network connection
+    const network = await provider.getNetwork();
+    if (network.chainId !== CHAIN_ID) {
+      throw new Error(
+        `Provider connected to wrong network. Expected ${CHAIN_ID}, got ${network.chainId}`,
+      );
+    }
+  });
+
+  test.skip('should sign a user operation', async () => {
```

Comment:
Can we unskip this test now? Is it possible for the integration test to actually run?